docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/git-open-pr.md

```diff
@@ -10,15 +10,15 @@ specified source and target branches. This step is often used after a
 [`git-push` step](git-push.md) and is commonly followed by a
 [`git-wait-for-pr` step](git-wait-for-pr.md).
 
-At present, this feature only supports GitHub and Gitea pull requests, and
-GitLab merge requests.
+At present, this feature only supports GitHub, Gitea and Azure DevOps pull
+requests, and GitLab merge requests.
```

Comment:
It always bugs me that GitLab calls them merge requests.

I think as this list grows, the "and GitLab merge requests" at the end feels more and more awkward.

What would you think about this?

```suggestion
At present, this feature only supports GitHub, Gitea, Azure DevOps, and
GitLab pull/merge requests.
```

LGTM either way.